drivers/scheduler/k8s/k8s.go

```diff
@@ -4983,6 +5010,36 @@ func (k *K8s) createAdmissionRegistrationObjects(
 	return nil, nil
 }
 
+// createVirtualMachineObjects creates the kubevirt VirtualMachines using kubectl apply
+func (k *K8s) createVirtualMachineObjects(
+	spec interface{},
+	ns *corev1.Namespace,
+	app *spec.AppSpec,
+) (interface{}, error) {
+	var specDir string
+	if obj, ok := spec.(*kubevirtv1.VirtualMachine); ok {
+		log.Warnf("Applying VirtualMachine spec for app [%s]", app.Key)
+		// This sleep is added in order to wait for the PVC to get bound.
+		// If this is not provided we are seeing errors like this when VM is starting:
+		// preparing host-disks failed: unable to create /var/run/kubevirt-private/vmi-disks/disk0/disk.img, not enough space
+		time.Sleep(30 * time.Second)
```

Comment:
Why can't we add a check to make sure pvc is in bout state instead of going with this sleep directly?